src/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -493,16 +504,18 @@ export class MongoEntityConverter {
     }
   }
 
-  public convertToDevice(mongoDevice: MongoDevice): Device {
+  public convertToDeviceInterface(mongoDevice: MongoDevice): Device {
```

Comment:
We no longer have anything called `DeviceInterface` so this method (and the one similar below) should have the `interface` part of their name removed again.